protocol/x/vault/keeper/orders.go

```diff
@@ -30,10 +32,79 @@ const (
 )
 
 // RefreshAllVaultOrders refreshes all orders for all vaults by
-// TODO(TRA-134)
 // 1. Cancelling all existing orders.
 // 2. Placing new orders.
 func (k Keeper) RefreshAllVaultOrders(ctx sdk.Context) {
+	// Iterate through all vaults.
+	totalSharesIterator := k.getTotalSharesIterator(ctx)
+	defer totalSharesIterator.Close()
+	for ; totalSharesIterator.Valid(); totalSharesIterator.Next() {
+		var vaultId types.VaultId
+		k.cdc.MustUnmarshal(totalSharesIterator.Key(), &vaultId)
+		var totalShares types.NumShares
+		k.cdc.MustUnmarshal(totalSharesIterator.Value(), &totalShares)
+
+		// Skip if TotalShares is non-positive.
+		if totalShares.NumShares.Cmp(dtypes.NewInt(0)) <= 0 {
+			continue
+		}
+
+		// Refresh orders depending on vault type.
+		switch vaultId.Type {
+		case types.VaultType_VAULT_TYPE_CLOB:
```

Comment:
this is the only vault type we support currently right? 

nit: add comment that no other vault types exist

Reply:
nit: Add a default case and error there.